mypy/semanal_pass1.py

```diff
@@ -124,6 +124,11 @@ def visit_file(self, file: MypyFile, fnam: str, mod_id: str, options: Options) -
                     # Maybe add 'bool' to all fixtures?
                     literal_types.append(('True', AnyType(TypeOfAny.special_form)))
 
+                # We are running tests without 'list' in builtins.
+                # TODO: Maybe optionally choose an default fixture stub per test file?
+                if 'list' not in self.sem.globals:
+                    literal_types.append(('list', AnyType(TypeOfAny.special_form)))
```

Comment:
This is very dangerous. With this addition many tests will pass (of fail in ultra-weird way) if someone forgets to add a fixture with `list`. It is OK with _literals_, but not with a class.

Reply:
Isn't `list` a literal?